doc/contributing/collaborator-guide.md

```diff
@@ -437,6 +437,16 @@ For pull requests introducing new core modules:
 * Land with a [Stability Index][] of Experimental. The module must remain
   Experimental until a semver-major release.
 
+### Introducing new APIs on the global scope
+
+Exposing APIs to the global scope unconditionally without any CLI flags must
```

Comment:
I think you should clarify that this includes properties of global objects defined by standards, e.g. `Symbol`.